src/HID-APIs/AbsoluteMouseAPI.hpp

```diff
@@ -80,6 +80,7 @@ void AbsoluteMouseAPI::moveTo(int x, int y, signed char wheel){
 	report.buttons = _buttons;
 	// The range -32768...32767 is converted to 0...32767 because Windows 7
 	// does not support negative coordinates.
+	// Negative values are supported here for compatibility reasons.
```

Comment:
Well it is not only compatibility, it is the API. I wanted an API that works with 0,0 being in the center. "Nobody" cares about the underlying usb part, so this comment is not correct. I have no better idea how to describe it though

Reply:
I tried to improve it